changelog.d/2-features/open-telemetry-instrumentation-brig-galley-gundeck-cannon

```diff
@@ -0,0 +1 @@
+added open telemetry instrumentation for brig, galley, gundeck and cannon
```

Comment:
maybe explain the consequences? where is the data going? how can i disable it going anywhere?

Reply:
this info should also go into the markdown docs.